packages/core/src/index.ts

```diff
@@ -45,3 +45,6 @@ export * from './geometry/Attribute';
 export * from './geometry/Buffer';
 export * from './geometry/Geometry';
 export * from './geometry/ViewableBuffer';
+
+// T stands for (content) type, L stands for the array fixed length
+export type ArrayFixed<T, L extends number> = [ T, ...Array<T> ] & { length: L };
```

Comment:
Can we move this to something like **packages/utils/src/types/**, seems like a more appropriate place for these generic types

Reply:
should be in utils, yes.

Reply:
Ok it's been added there now